scaleway/helpers_lb.go

```diff
@@ -309,9 +310,10 @@ func expandLbHCHTTP(raw interface{}) *lbSDK.HealthCheckHTTPConfig {
 	}
 	rawMap := raw.([]interface{})[0].(map[string]interface{})
 	return &lbSDK.HealthCheckHTTPConfig{
-		URI:    rawMap["uri"].(string),
-		Method: rawMap["method"].(string),
-		Code:   expandInt32Ptr(rawMap["code"]),
+		URI:        rawMap["uri"].(string),
+		Method:     rawMap["method"].(string),
+		Code:       expandInt32Ptr(rawMap["code"]),
+		HostHeader: rawMap["host_header"].(string),
```

Comment:
Why this is keep it as `string`?  https://pkg.go.dev/github.com/scaleway/scaleway-sdk-go@v1.0.0-beta.13/api/lb/v1#HealthCheckHTTPConfig

Reply:
I don't understand, it's a [string](https://pkg.go.dev/github.com/scaleway/scaleway-sdk-go@v1.0.0-beta.13/api/lb/v1#HealthCheckHTTPConfig) ?